modules/store/spec/feature_creator.spec.ts

```diff
@@ -98,6 +104,104 @@ describe('createFeature()', () => {
     });
   });
 
+  describe('derived selectors', () => {
+    it('should create a feature', () => {
+      const initialState = { count1: 9, count2: 10 };
+      const counterFeature = createFeature({
+        name: 'counter',
+        reducer: createReducer(initialState),
+        derivedSelectors: ({
```

Comment:
Thoughts on the `derivedSelectors` property name? It was `selectors` in the initial proposal.

Reply:
I think `additionalSelectors` or `extraSelectors` would be more clear. Could some of the selectors added to the group not be derived, but just grouped under the feature?

Reply:
Great idea! I like `extraSelectors`. It reminds me of `extraReducers` from RTK `createSlice` config.

Reply:
+1 for `extraSelectors`